config/locales/en/errors.yml

```diff
@@ -50,3 +50,4 @@ en:
   staff_response:
     errors:
       must_reject_or_accept_visit: "A visit must either be accepted by choosing a date or at least one rejection reason"
+      lead_visitor_not_on_contact_list: The lead visitor is not on the contact list
```

Comment:
@StupidCodeFactory @kyliehavelock @chrishanes Think this should be covered by the generic rejection message we're going to put in: "This visit will be rejected and an email will be sent to the lead visitor"